tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/DependencyResolutionTests.fs

```diff
@@ -18,4 +21,4 @@ let ``Supported scenario`` (scenario: Scenario) =
     for file in scenario.Files do
         let expectedDeps = file.ExpectedDependencies
         let actualDeps = set graph.[file.Index]
-        Assert.AreEqual(expectedDeps, actualDeps, $"Dependencies don't match for {System.IO.Path.GetFileName file.FileName}")
+        Assert.True((expectedDeps = actualDeps), $"Dependencies don't match for {System.IO.Path.GetFileName file.FileName}")
```

Comment:
There is no overload of `Assert.Equal` with a message, this one is rather useful, so keeping it in this shape.